addons/point_of_sale/static/src/app/screens/product_screen/product_info_popup/product_info_popup.js

```diff
@@ -22,7 +22,7 @@ export class ProductInfoPopup extends Component {
         return isAccessibleToEveryUser || isCashierManager;
     }
     editProduct() {
-        this.pos.editProduct(this.props.product);
+        this.pos.editProduct(this.props.productTemplate.product_variant_ids[0]);
```

Comment:
I'm okay with the fix as I can see the same behavior on the product info popup.

https://github.com/odoo/odoo/blob/master/addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js#L456